frontend/src/citizen-frontend/map/MapView.tsx

```diff
@@ -96,12 +115,14 @@ export default React.memo(function MapView() {
   useTitle(t, t.map.title)
 
   return (
-    <FullScreen data-qa="map-view">
+    <MapFullscreenContainer loggedIn={loggedIn}>
```

Comment:
![Screenshot 2021-09-28 at 12 05 10](https://user-images.githubusercontent.com/1533854/135057848-c70be89b-7c1f-4084-bbb6-f191e1e74f00.png)
Something wrong with the map. Sometimes that gray area goes away when moving the map but reappears on refresh. No issue when logged out.